domains/transfer/Shared/ProjectOriginClients.Tests/ProjectOriginWalletClientTests.cs

```diff
@@ -25,7 +25,7 @@ public async Task CreateAndGetWallets()
     {
         var ownerSubject = Guid.NewGuid();
         var httpClient = GetWalletHttpClient();
-        var walletClient = new ProjectOriginWalletClient(httpClient);
+        var walletClient = new ProjectOriginWalletClient(httpClient, null); // Do we want to test Proxy here?
```

Comment:
Should we make a Proxy Stack? But proxy is reliant to Wallet?
Or perhaps just  Wiremock? Otherwise we are testing dependencies of dependencies.